### PR TITLE
benchmark: add header with meta information

### PR DESCRIPTION
**Pull request by igoropaniuk**
`Signed-off-by: Igor Opaniuk <igor.opaniuk@linaro.org>`

### CODE REVIEW

**Comment by jockebech** [comment]
I have no prior experience with the yaml code in c, but the code as such looks good.
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: main.c [hunk 1]
```diff
@@ -205,6 +224,19 @@ static bool init_emitter(FILE *ts_file)
...
 		ERROR_GOTO(emitter_delete,
 				"Failed to emit YAML sequence mapping event");
+	/*
+	 * Filling header
+	 */
+	snprintf(data, MAX_SCALAR, "%" PRIu64, bench_ts_global->cores);
+	fill_map("cores", data);
+
+	snprintf(data, MAX_SCALAR, "%" PRIu64, bench_ts_global->freq);
+	fill_map("timer_frequency", data);
```
Comment (etienne-lms) [inline main.c]: Return with error on failure ?
The same comment could apply on calls to `fill_map()` from `fill_timestamp()`.

Comment (igoropaniuk) [reply main.c]: @etienne-lms agree, will fix

Comment (igoropaniuk) [reply main.c]: @etienne-lms fixed

**Pull request closed without merging**